src/applications/personalization/profile/components/personal-health-care-contacts/PersonalHealthCareContacts.jsx

```diff
@@ -0,0 +1,131 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import {
+  selectProfileContactsToggle,
+  selectProfileContacts,
+  selectEmergencyContact,
+  selectNextOfKin,
+} from '@@profile/selectors';
+import { fetchProfileContacts } from '@@profile/actions';
+import { focusElement } from '~/platform/utilities/ui';
+
+import Contact from './Contact';
+import Loading from './Loading';
+import LoadFail from '../alerts/LoadFail';
+
+const HelpDeskContact = () => (
+  <>
+    <va-telephone data-testid="va-800-number" contact="8006982411" /> (
+    <va-telephone data-testid="va-711-number" contact="711" tty="true" />)
+  </>
+);
+
+const PersonalHealthCareContacts = () => {
+  const dispatch = useDispatch();
+  const { data, loading, error } = useSelector(selectProfileContacts);
+  const { featureToggles } = useSelector(state => state);
+  const enabled = useSelector(selectProfileContactsToggle);
+  const emergencyContact = useSelector(selectEmergencyContact);
+  const nextOfKin = useSelector(selectNextOfKin);
+
+  useEffect(
+    () =>
+      enabled &&
+      !data &&
+      !loading &&
+      !error &&
+      dispatch(fetchProfileContacts()),
+    [data, dispatch, enabled, error, loading],
+  );
+
+  useEffect(() => {
+    document.title = 'Personal Health Care Contacts | Veterans Affairs';
+  }, []);
+
+  useEffect(() => enabled && !loading && focusElement('[data-focus-target]'), [
+    enabled,
+    loading,
+  ]);
+
+  if (!enabled) return <></>;
+  if (featureToggles.loading || loading) return <Loading />;
+  if (error) return <LoadFail />;
+
+  return (
+    <div className="vads-u-padding-bottom--4">
+      <h1
+        tabIndex="-1"
+        className="vads-u-font-size--h2 vads-u-margin-y--2 medium-screen:vads-u-margin-bottom--4 medium-screen:vads-u-margin-top--3"
+        data-focus-target
+      >
+        Personal health care contacts
+      </h1>
+
+      <div className="vads-u-margin-bottom--3">
+        <va-additional-info
+          className=""
+          data-testid="phcc-how-to-update"
+          trigger="Learn how to update your personal health care contact information"
+          uswds
+        >
+          <p>
+            If this information isn’t correct, here’s how to update it:
+            <ul className="vads-u-margin-y--0">
+              <li>Ask a staff member at your next appointment, or</li>
+              <li>
+                Call us at <HelpDeskContact />. We’re here 24/7.
+              </li>
+            </ul>
+          </p>
```

Comment:
There is a warning that is logged when running this locally:

> Warning: validateDOMNesting(...): <ul> cannot appear as a descendant of <p>.

I'd suggest wrapping the whole thing in a div since there is an issue with styling without it (I'll see if I can find the issue around that) and then probably just having the first section be a <p> and the <ul> be its own thing.